modules/eco-poison-pill-operator-about.adoc

```diff
@@ -67,15 +67,15 @@ The `PoisonPillRemediationTemplate` CR resembles the following YAML file:
 apiVersion: poison-pill.medik8s.io/v1alpha1
 kind: PoisonPillRemediationTemplate
 metadata:
-creationTimestamp: "2022-03-02T08:02:40Z"
-generation: 1
-name: poison-pill-default-template
-namespace: openshift-operators
-resourceVersion: "596469"
-uid: 5d29e437-c485-48fa-ba9e-0354649afd31
-spec:
-template:
+  creationTimestamp: "2022-03-02T08:02:40Z"
+  generation: 1
+  name: poison-pill-default-template
+  namespace: openshift-operators
+  resourceVersion: "596469"
+  uid: 5d29e437-c485-48fa-ba9e-0354649afd31
 spec:
-remediationStrategy: NodeDeletion <1>
+  template:
+    spec:
+      remediationStrategy: NodeDeletion (1)
```

Comment:
I'm surprised it works as is (at least in the preview), but the correct syntax for the callout here is `<1>`. Would you mind updating it? I'm afraid it's possible that the customer portal might not render it properly.

Reply:
Good catch, @bergerhoffer! I have updated it now.